File: Lv.1/문자열 다루기 기본/solution.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12918
function solution(s) {
  return (s.length === 4 || s.length === 6) && s.match(/[a-z]|[A-Z]/g) === null
    ? true
    : false;
}

File: Lv.1/문자열 다루기 기본/solution.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12918?language=python3
import re

def solution(s):    
    return True if re.search(r'^\d{4}$|^\d{6}$', s) else False

# isdigit 함수가 있ㅜㅏ.
# def alpha_string46(s):
#     return s.isdigit() and len(s) in (4, 6)